python/packages/autogen-agentchat/src/autogen_agentchat/base/_task.py

```diff
@@ -23,7 +23,7 @@ class TaskRunner(Protocol):
     async def run(
         self,
         *,
-        task: str | ChatMessage | None = None,
+        task: str | ChatMessage | List[ChatMessage] | None = None,
```

Comment:
Need to update `BaseChatAgent` which also implements the `TaskRunner` protocol.

Reply:
Updated the type hints for both `run` and `run_stream` methods to include `List[ChatMessage]`